lint/action.yaml

```diff
@@ -64,7 +64,13 @@ runs:
       env:
         NPM_AUTH_TOKEN: ${{ inputs.npm-auth-token }}
         NPM_TOKEN: ${{ inputs.npm-token }}
+    - name: Check for .pre-commit-config.yaml file
+      id: check_pre_commit_config
+      uses: andstor/file-existence-action@v2
+      with:
+        files: .pre-commit-config.yaml
```

Comment:
Should you account for `yml` extension?

Reply:
Or any other format `.pre-commit` config files could use?

Reply:
what other files _can_ it use?

Reply:
I'm not sure I think it's just yaml files according to hteir docs https://pre-commit.com/